docs/guides/manage-cost.mdx

```diff
@@ -83,7 +106,7 @@ You can view your billing cycle on the [Billing and usage page](https://cloud.ib
 
 ### View job cost
 
-To determine how much has been billed to each job associated with an instance, from the [Instances page](https://cloud.ibm.com/quantum/instances), click the instance to open its details page. Next, on the left side, click Jobs.
+To determine how much has been billed to each job associated with an instance, from the <LegacyContent>[IBM Cloud Instances page](https://cloud.ibm.com/quantum/instances)</LegacyContent><CloudContent>[Instances page](https://quantum.cloud.ibm.com/instances)</CloudContent>, click the instance to open its details page. Next, on the left side, click Jobs.
```

Comment:
Same here. We are not showing this information now. This is an screenshot of what users see now:
<img width="1038" alt="Captura de pantalla 2025-02-28 a las 18 22 57" src="https://github.com/user-attachments/assets/6e1e350b-9d2f-4ab4-8d83-2283adb3d8ec" />